### DISCUSSION TITLE
Which token do we need to use ?

### DISCUSSION THREAD

**Discussion by frderoubaix** [Lucky3028/terraform-provider-discord#139] [Q&A] [answered]
Hello,
I try to create my own server with a terraform script, but I have this message : 
```
 Error: Failed to create server: HTTP 401 Unauthorized, {"message": "401: Unauthorized", "code": 0}
│
│   with discord_server.cron-as-a-service-discord-server,
│   on [discord.tf](http://discord.tf/) line 10, in resource "discord_server" "cron-as-a-service-discord-server":
│   10: resource "discord_server" "cron-as-a-service-discord-server" {
```

I think my Token was wrong, but how to get the good token.
Maybe we can't directly create a server and just make action on existing server ? 

Thanks in advance for your answer.

**Comment by myroslav** [comment, marked as answer]
See #140 for examples.

To summarize, prefixing your token with "`Bot `" should resolve the issue. This is a peculiar requirement of the [bwmarrin/discordgo](https://github.com/bwmarrin/discordgo) library, which is currently utilized by version 1.7.0 of the provider.

**Comment by frderoubaix** [comment]
Oh ok ! thanks for all the informations !
